src/client/CourseRequestForm/index.js

```diff
@@ -59,11 +59,11 @@ const CourseRequestForm = ({courses}) => {
             <Table.Cell>
                 <Flex justifyItems='center'>
                     <Flex.Item>
-                        <Checkbox label='' size='small' defaultChecked/>
```

Comment:
@kubilayagi Can you get the wheter or not to have this checked by default from the config file?